proto/rill/runtime/v1/resources.proto

```diff
@@ -495,10 +495,18 @@ message API {
 message APISpec {
   string resolver = 1;
   google.protobuf.Struct resolver_properties = 2;
+  OpenAPISpec open_api_spec = 3;
```

Comment:
Consider just inlining the values here, for example:
```
string openapi_summary = 3;
repeated google.protobuf.Struct openapi_parameters = 4;
repeated google.protobuf.Struct openapi_response_schema = 5;
```

Nested types are nice in YAML, but less so in protos. And especially the name `OpenAPISpec` could be confused for a resource named "OpenAPI", as opposed to a sub-type of `APISpec`.